subcommands/keys/tuf_rotate_offline_key.go

```diff
@@ -206,12 +223,29 @@ func doRotateOfflineKey(cmd *cobra.Command, args []string) {
 		// 2. Generate a new key.
 		// 3. Set these keys in root.json.
```

Comment:
I guess it should be `add this key to` in the step #3 since the step #2 generates a key not keys?

Reply:
Ah, OK, when reviewing the commit change the overall function is not visible.